src/components/custom-sdk/template/HMRC_ODX_GDSCheckAnswersPage/index.tsx

```diff
@@ -100,13 +100,18 @@ export default function HmrcOdxGdsCheckAnswersPage(props: HmrcOdxGdsCheckAnswers
       invalidateCache: true
     };
 
+    const dataPageName = isEduStartJourney()
+      ? 'D_GetCYAStepIDByApplication'
+      : 'D_GetCurrentCYAStepID';
+
     PCore.getDataPageUtils()
```

Comment:
Should we add any error handling here what will happen if the "PC.getPageUtils()" fails?
ie 
try {
  PCore.getDataPageUtils()
  ...Rest of code...
  } catch (error) {
// log error, or display error message
}

Reply:
Yes, look like on line  128 we just console log the error. Could this be improved?

Reply:
Yes but we need error handling mechanism for whole app. Any suggestion?

Reply:
Error boundaries are usually used. But probably out of scope for this user story